test/tools/ossfuzz/SolidityMutator.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <test/tools/ossfuzz/SolidityMutator.h>
+
+#include <libsolutil/Whiskers.h>
+
+#include <boost/algorithm/string/join.hpp>
+
+#include <algorithm>
+
+using namespace std;
+using namespace solidity::util;
+using namespace solidity::test::fuzzer;
+
+string SolidityMutator::genRandString(unsigned int _length, char _start, char _end)
+{
+	uniform_int_distribution<char> dist(_start, _end);
+	string result{};
+	generate_n(back_inserter(result), _length, [&]{ return dist(Rand); });
+	return result;
+}
+
+antlrcpp::Any SolidityMutator::visitPragmaDirective(SolidityParser::PragmaDirectiveContext*)
+{
+	auto PickLiteral = [](unsigned const _idx) -> string
+	{
+		static const vector<string> alphanum = {
+			"0",
+			"experimental",
+			"solidity < 142857",
+			"solidity >=0.0.0",
+			"solidity >=0.0.0 <0.8.0",
+			"experimental __test",
+			"experimental SMTChecker",
+			"experimental ABIEncoderV2",
+			"experimental \"xyz\""
+		};
+
+		return alphanum[_idx % alphanum.size()];
+	};
+
+	string pragma{};
+	if (coinFlip())
+		pragma = PickLiteral(Rand());
+	else
+	{
+		// Generate a random pragma that does not contain a semicolon
+		pragma = genRandString(randOneToN(30), '<');
```

Comment:
It would be a bit more readable without relying on the default. Otherwise it's easy to assume that it will just generate a random number of `<` characters.

Also, if `<` was chosen as the first ASCII character after the semicolon then I think that `';' + 1` would be more self-explanatory.
```suggestion
		pragma = genRandString(randOneToN(30), ';' + 1, '~');
```